skimage/restoration/j_invariant.py

```diff
@@ -234,17 +248,20 @@ def calibrate_denoiser(image, denoise_function, denoise_parameters, *
     best_parameters = parameters_tested[idx]
 
     best_denoise_function = functools.partial(
-        invariant_denoise,
+        _invariant_denoise,
         denoise_function=denoise_function,
         stride=stride,
         multichannel=multichannel,
         denoiser_kwargs=best_parameters,
     )
 
-    return best_denoise_function
+    if full_output:
+        return best_denoise_function, parameters_tested, losses
```

Comment:
here we would rather return `best_denoise_function, (parameters_tested, losses)`